ui/lib/core/addon/components/form-field.hbs

```diff
@@ -16,22 +16,41 @@
   {{/unless}}
   {{#if @attr.options.possibleValues}}
     {{#if (eq @attr.options.editType "radio")}}
-      <div class="control columns" data-test-radio-input>
+      <div class="control {{unless this.columnRadio 'columns'}}" data-test-radio-input>
         {{#each (path-or-array @attr.options.possibleValues @model) as |val|}}
-          <div class="column is-narrow is-flex-center has-text-grey has-right-margin-s" data-test-input={{@attr.name}}>
+          <div
+            class="is-flex-center
+              {{if
+                this.columnRadio
+                'has-top-padding-xs has-bottom-padding-s'
+                'column is-narrow has-text-grey has-right-margin-s'
+              }}"
+            data-test-input={{@attr.name}}
+          >
             <RadioButton
               class="radio"
               name={{@attr.name}}
               id={{or val.value val}}
               value={{or val.value val}}
-              @value={{or val.value val}}
+              @value={{if (or (eq @attr.type "boolean") val.value) val.value val}}
               @onChange={{this.setAndBroadcast}}
               @groupValue={{(get @model this.valuePath)}}
               @disabled={{and @attr.options.editDisabled (not @model.isNew)}}
+              data-test-radio="{{if (or (eq @attr.type 'boolean') val.value) val.value val}}"
             />
-            <label for={{or val.value val}} class="has-left-margin-xs">
-              {{or val.value val}}
-            </label>
+            <div class="has-left-margin-xs">
+              <label
+                for="{{or val.value val}}"
+                class="has-left-margin-xs {{if this.columnRadio 'has-text-black is-size-7'}}"
+              >
+                {{or val.label val.value val}}
+                {{#if this.columnRadio}}
```

Comment:
this getter is a little confusing - the styling change makes it so the label has subtext that renders beneath it, right? 

I think renaming to `radioHelperText` - I know I mentioned using HDS verbiage here. But I'm wondering if we should refer to this a subtext still since that's how we do it in this component. 

That way when we do the form revamp we know subText = HDS helpertext and our current "helpText" means an info tooltip